Packs/PassiveTotal/Integrations/integration-PassiveTotal.yml

```diff
@@ -562,11 +570,13 @@ script:
       description: The actual score
     description: Get malware data
   - name: url
+    deprecated: true
     arguments:
     - name: url
       required: true
       default: true
       description: The URL to check reputation for
+      isArray: true
```

Comment:
Why adding this if deprecating the integration?

Reply:
because of `demisto-sdk format` command